<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1643</width>
    <height>1014</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FIT4-5仿真平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{background-color: rgb(247, 247, 247); color:rgb(51,51,51); border: 1px solid  rgb(247, 247, 247);outline:0px;}
QListWidget::Item{background-color: rgb(247, 247, 247);}
QListWidget::Item:hover{background-color: rgb(247, 247, 247); }
QListWidget::item:selected{
	background-color: rgb(247, 247, 247);
	color:black; 
    border: 1px solid  rgb(247, 247, 247);
}
QListWidget::item:selected:!active{border: 1px solid  rgb(247, 247, 247); background-color: rgb(247, 247, 247); color:rgb(51,51,51); } </string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cleartrackbutton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>890</y>
      <width>1151</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:6px;
padding:2px 4px;
border-style: outset;
border:2px groove gray;
background-color: rgb(229, 234, 233);
color: rgb(37, 194, 118);</string>
    </property>
    <property name="text">
     <string>Clear the Track</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="map_view_mini">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="map_view_real">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>1141</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1643</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_map"/>
    <addaction name="actionsave_Tag_Location"/>
    <addaction name="actionsave_mapView"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionTag_location"/>
    <addaction name="actionadd_preson"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionshowRoomLabel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open_map">
   <property name="text">
    <string>&amp;open_map</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;exit</string>
   </property>
  </action>
  <action name="actionTag_location">
   <property name="text">
    <string>tag_location</string>
   </property>
  </action>
  <action name="actionadd_preson">
   <property name="text">
    <string>add_preson</string>
   </property>
  </action>
  <action name="actionsave_Tag_Location">
   <property name="text">
    <string>save_Tag_Location</string>
   </property>
  </action>
  <action name="actionsave_mapView">
   <property name="text">
    <string>save_mapView</string>
   </property>
  </action>
  <action name="actionshowRoomLabel">
   <property name="text">
    <string>showRoomLabel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
